shared/cluster.go

```diff
@@ -336,6 +336,22 @@ func GetPods(print bool) ([]Pod, error) {
 	return pods, nil
 }
 
+// GetPodsByNamespaceAndLabel returns pods parsed from kubectl get pods in a specific namespace with a specific label
+func GetPodsByNamespaceAndLabel(namespace, label string, print bool) ([]Pod, error) {
```

Comment:
do we have a need to print this ?

Reply:
Nah but was keeping it similar to the `GetPods` function that already exists and has print capability. I actually don't love all the prints so I am more than happy to remove it 😄

Reply:
lol

but the getPods its pretty useful despite not liking as well

thats why i was wondering maybe this one is useful also